news/2024/2024-09-18-osumania-4k-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,139 @@
+
+- [trapman - minefield outlaw](https://osu.ppy.sh/beatmapsets/2246339#mania/4775913) ([guden](https://osu.ppy.sh/users/11626065), RC4) @ [13:53](https://www.twitch.tv/videos/2250787141?t=00h13m53s) - 1–0, Hong Kong’s turn to pick, and their pick was RC4, a stamina-draining chordjack map with tricky patterns waiting to break combos. While Hong Kong was able to stay in the lead for a good chunk of the map, a few combo breaks near the end of the map let the Philippines take the break point.
+- [YOASOBI - Idol](https://osu.ppy.sh/beatmapsets/2246343#mania/4775926) ([0DZ0](https://osu.ppy.sh/users/28156707), RC6) @ [18:29](https://www.twitch.tv/videos/2250787141?t=00h18m29s) - Hong Kong needed revenge from the Philippines if they wanted to stay alive, and in an interesting turn of events, the Philippines chose RC6, a vocal stream dump map. This pick ended up being way too close with a score difference not even 10k, and in the end, Hong Kong managed to respond in kind with a break point.
+- [Tatsh - Joker -Vocaloid ver.-](https://osu.ppy.sh/beatmapsets/2246033#mania/4775287) ([Valedict](https://osu.ppy.sh/users/2537082), RC1) @ [58:25](https://www.twitch.tv/videos/2250787141?t=00h58m25s) - It's now match point for the Philippines at 6–3. It was time for Hong Kong to make their pick, and their pick was RC1, a dense stream map with practically zero breaks. Hong Kong needed to win if they wanted to continue having a fighting chance at taking this to the tiebreaker. All was going well, until the combos broke from the Hong Kong side at the end, resulting in the Philippines winning the match.
+
+Well... that was certainly a match. However, we will have to say our goodbyes to the Hong Kong team as they are now officially out of the tournament. Your underrated performances that shocked many viewers will not be forgotten, and we hope for a better run next year. As for the Philippines, they will be facing up against the almighty South Korean team which has previously dominated the osu!mania World Cup. We wish them the very best of luck.
+
+### ::{ flag=JP }:: Japan vs. ::{ flag=CN }:: China
+
+[**You can view the match information here!**](https://osu.ppy.sh/community/matches/115504471)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2251756726&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+In 2022's MWC 4K , China was beaten by Japan in Semifinals, and by coincidence, they met in same round again this year. This time both teams changed a lot, China just swept Vietnam in round 1, while Japan successfully defeated Singapore without much trouble. It is well-known that Japan has a strong roster with impressive regular skill, and China players shows their great consistency in every skill set. Both teams are also strong in SV, and that ended up being the key to winning this match.
```

Comment:
```suggestion
In 2022's MWC 4K, China was beaten by Japan in the Semifinals, and by coincidence, they met in same round again this year. Both teams have changed a lot since: China just swept Vietnam in round 1, while Japan successfully defeated Singapore without much trouble. It is well known that Japan has a strong roster with impressive raw skill, and that Chinese players have great consistency in every skill set. Both teams are also strong in SV, and that ended up being the key to winning this match.
```